### PR TITLE
Added-Feature-desktop-version

### PR DESCRIPTION
**Pull request by kinginthenorthcodez**
#Features
-Added navigations links
-Added media queries for responsive layout

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -461,6 +488,317 @@ textarea:focus {
...
+    margin: 0 auto;
+    position: relative;
+    right: 50px;
+    flex: none;
+    margin-bottom: 20px;
+    background-image: url("../images/illustration-1-contact-form-desktop.png");
+    background-size: contain;
+    background-repeat: no-repeat;
+    background-position: left top;
+  }
```
Comment (Whoistolu) [inline assets/css/style.css:749]: - [ ] [REQUIRED] Kindly ensure that the content of your form-container class has equal margins on the right and left. Right now the right margin on your form is much than the left margin. Please check the figma template to get the exact margins.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
 |

File: assets/css/style.css [hunk 2]
```diff
@@ -461,6 +488,317 @@ textarea:focus {
...
+  .work-card-title {
+    width: 337px;
+    height: 57px;
+  }
+
+  .about-headline {
+    margin: 24px 0;
+    position: relative;
+    left: 400px;
+  }
```
Comment (Whoistolu) [inline assets/css/style.css:687]: - [ ] [REQUIRED] Kindly ensure that you stick to the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468) as much as possible (font, font-size, margin, padding, color).

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
 | 

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
 |

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @kinginthenorthcodez ,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

_Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

#### N/A

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: assets/css/style.css (hunk 2 above)
Comment (kinginthenorthcodez) [reply assets/css/style.css]: Updated fixes for the desktop design

File: assets/css/style.css (hunk 1 above)
Comment (kinginthenorthcodez) [reply assets/css/style.css]: fixed errors

File: assets/css/style.css (hunk 2 above)
[the remark this answers is not in the document]
Comment (kinginthenorthcodez) [reply assets/css/style.css]: Fixed about me section 👍🏽

File: assets/css/style.css [hunk 3]
```diff
@@ -357,6 +365,25 @@ ul > li {
   box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
```
[the remark this answers is not in the document]
Comment (kinginthenorthcodez) [reply assets/css/style.css:371]: changes made according to the review 👍🏽

**Review by alexander16108 — APPROVED** [review]
Hi @kinginthenorthcodez, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**